app/services/amr/csv_parser.rb

```diff
@@ -0,0 +1,45 @@
+require 'csv'
+
+module Amr
+  class CsvParser
```

Comment:
If we have services in the services folder, do we want to call them blah_service? or just blah? I think we have a mix of the two at the moment.... Happy to go either way